demisto_sdk/commands/validate/validators/PB_validators/PB115_is_tasks_quiet_mode.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.content_graph.parsers.related_files import RelatedFileType
+from demisto_sdk.commands.content_graph.objects.playbook import Playbook
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = Playbook
+
+
+class IsTasksQuietModeValidator(BaseValidator[ContentTypes]):
+    error_code = "PB115"
+    description = "Checks if all tasks in a playbook are in quiet mode."
+    error_message = "Playbook '{playbook_name}' contains tasks that are not in quiet mode (quietmode: 2)."
+    fix_message = "Fixed playbook to set all tasks with (quietmode: 2) to (quietmode: 0)."
+    related_field = "tasks"
+    is_auto_fixable = True
+    related_file_type = [RelatedFileType.YML]
+
+    def is_valid(self, content_items: Iterable[ContentTypes]) -> List[ValidationResult]:
+        """
+        Validates that tasks in content_items(playbook) are in quiet mode if they contain an input query for "indicators".
+
+        Args:
+            content_items (Iterable[ContentTypes]): Content items to validate.
+
+        Returns:
+            List[ValidationResult]: Validation results for items not meeting criteria.
+        """
+        return [
+            ValidationResult(
+                validator=self,
+                message=self.error_message,
+                content_object=content_item,
+            )
+            for content_item in content_items
+            if (
+                any(
+                    (i.get("playbookInputQuery") or {}).get("queryEntity") == "indicators"
+                    for i in content_item.data.get("inputs", {})
+                )
+                and [
+                    task_key
+                    for _, task_key in content_item.tasks.items()
+                    if task_key.quietmode == 2
+                ]
```

Comment:
Let's return the ids of the tasks that uses quite mode to make the message error easier for the user to fix.
Also, let's save the tasks so we can reuse it in the fix function.
See Sapir's [pr](https://github.com/demisto/demisto-sdk/pull/4360/files#diff-789dba8ca3e318b657b72a8662bf27bbb0a162501791238c7410db2bedeab72f) as a good reference.